infra/test/integration/reviewsite/serverspec/default_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'serverspec'
+
+USER = "vagrant"
+WORKSPACE = "/home/#{USER}/workspace"
+PATH = "/opt/rbenv/shims:/opt/rbenv/bin:/opt/rbenv/plugins/ruby_build/bin:$PATH";
+BUNDLE_PATH = "~/bundle"
+
+# Required by serverspec
+set :backend, :exec
+set :path, PATH
+
+RSpec.configure do |c|
```

Comment:
Unused block argument - `c`. You can omit the argument if you don't care about it.